Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "concordium-rosetta"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
If I wanted to bikeshed I would comment here that this would probably warrant a minor version bump :P

Reply:
But if you did then I would point out that pre-v1, anything goes ;) https://semver.org/#spec-item-4